tools/scoring/score/dimensions/entity_point_identification.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Core component """
+
+from score.dimensions.dimension import Dimension
+from score.types_ import DeserializedFilesDict, PointsVirtualList
+from score.constants import FileTypes
+
+PROPOSED, SOLUTION = FileTypes
+
+
+class EntityPointIdentification(Dimension):
+  """
+  Quantifies whether the proposed file
+  included the correct points in each entity.
+  """
+  def __init__(self, *, deserialized_files: DeserializedFilesDict):
```

Comment:
in general the `init` must not do operations. 
can you please create another function to do the operations?

Reply:
I'll have to change all dimensions — will do.